assets/airports/cyhz.json

```diff
@@ -0,0 +1,1347 @@
+                ["N45.334855900", "W63.318616100"],
+                ["N45.335457400", "W63.319512200"],
+                ["N45.336091100", "W63.320362300"],
+                ["N45.336754900", "W63.321163900"],
+                ["N45.337447200", "W63.321914900"],
+                ["N45.338165900", "W63.322613400"],
+                ["N45.338909200", "W63.323257400"],
+                ["N45.339675000", "W63.323845400"],
+                ["N45.340461200", "W63.324375700"],
+                ["N45.341265600", "W63.324847000"],
+                ["N45.342086100", "W63.325258000"],
+                ["N45.342920300", "W63.325607600"],
+                ["N45.343766300", "W63.325895000"],
+                ["N45.344621500", "W63.326119400"],
+                ["N45.345483700", "W63.326280200"],
+                ["N45.346350600", "W63.326377000"],
+                ["N45.347219800", "W63.326409500"],
+                ["N45.348089000", "W63.326377700"],
+                ["N45.348955900", "W63.326281600"],
+                ["N45.349818200", "W63.326121500"],
+                ["N45.350673500", "W63.325897800"],
+                ["N45.351519600", "W63.325611100"],
+                ["N45.352354000", "W63.325262000"],
+                ["N45.353174600", "W63.324851600"],
+                ["N45.353979200", "W63.324380800"],
+                ["N45.354765500", "W63.323851000"],
+                ["N45.355531600", "W63.323263400"],
+                ["N45.356275100", "W63.322619700"],
+                ["N45.356994100", "W63.321921400"],
+                ["N45.357686600", "W63.321170600"],
+                ["N45.358350700", "W63.320369100"],
+                ["N45.358984500", "W63.319519100"],
+                ["N45.359586400", "W63.318622900"],
+                ["N45.360154400", "W63.317682900"],
+                ["N45.360687200", "W63.316701700"],
+                ["N45.361183200", "W63.315681800"],
+                ["N45.361640900", "W63.314626100"],
+                ["N45.362059100", "W63.313537500"],
+                ["N45.362436600", "W63.312418900"],
+                ["N45.362772300", "W63.311273500"],
+                ["N45.363065200", "W63.310104400"],
+                ["N45.363314500", "W63.308914900"],
+                ["N45.363519500", "W63.307708200"],
+                ["N45.363679500", "W63.306487700"],
+                ["N45.363794200", "W63.305256800"],
+                ["N45.363863100", "W63.304019000"],
+                ["N45.363886100", "W63.302777800"]
+            ]
+        },
+        {
+            "name": "CYR748",
+            "height": "1500ft",
+            "coordinates": [
+                ["N44d36m06.00", "W063d26m06.00"],
+                ["N44d32m41.00", "W063d25m15.00"],
+                ["N44d26m54.00", "W063d10m48.00"],
+                ["N44d36m54.00", "W063d05m00.00"],
+                ["N44d38m00.00", "W063d24m30.00"],
+                ["N44d36m06.00", "W063d26m06.00"]
+            ]
+        }
+    ],
+    "runways": [
```

Comment:
The runway coordinates seem to be just slightly off, so the extended centerlines don't align perfectly with the fixes on the ILS

Reply:
I don't know the coordinates, maybe if you could send them to me please? @Fechulo